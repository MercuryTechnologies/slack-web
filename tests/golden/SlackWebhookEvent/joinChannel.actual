EventEventCallback
    ( EventCallback
        { eventId = EventId
            { unEventId = "Ev043W120WSH" }
        , teamId = TeamId
            { unTeamId = "T043DB835ML" }
        , eventTime = MkSystemTime
            { systemSeconds = 1664222685
            , systemNanoseconds = 0
            }
        , event = EventChannelJoinMessage
        }
    )