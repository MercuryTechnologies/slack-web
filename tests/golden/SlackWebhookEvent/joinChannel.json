{
  "token": "aaaa",
  "team_id": "T043DB835ML",
  "api_app_id": "A0442TUPHGR",
  "event": {
    "type": "message",
    "subtype": "channel_join",
    "ts": "1664222685.728809",
    "user": "U0442US8QGH",
    "text": "<@U0442US8QGH> has joined the channel",
    "inviter": "U043H11ES4V",
    "channel": "C043KSKGJUB",
    "event_ts": "1664222685.728809",
    "channel_type": "channel"
  },
  "type": "event_callback",
  "event_id": "Ev043W120WSH",
  "event_time": 1664222685,
  "authorizations": [
    {
      "enterprise_id": null,
      "team_id": "T043DB835ML",
      "user_id": "U0442US8QGH",
      "is_bot": true,
      "is_enterprise_install": false
    }
  ],
  "is_ext_shared_channel": false,
  "event_context": "aaaa"
}
